temporal-docker-builds.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: temporal-docker-builds
+  # This project doesn't do releases and everything is commit based.
+  # This corresponds to commit 70f8b8bbda3723fe7c8f822cc8aa379598ac8c86
+  version: 0.0_git20231116
+  epoch: 0
+  description: Temporal service Docker images build
+  copyright:
+    - license: MIT License
+
+vars:
+  commit: 70f8b8bbda3723fe7c8f822cc8aa379598ac8c86
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2eb296dc8493320261c762377356dedc23a4b3e4040502c21618e61b407a82a5
+      extract: "false"
+      uri: https://github.com/temporalio/docker-builds/archive/${{vars.commit}}.zip
+
+  - runs: |
+      unzip -q ${{vars.commit}}.zip
+      mv docker-builds-${{vars.commit}} docker-builds
+      rm *.zip
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/temporal-docker-builds
```

Comment:
Maybe put these files under `/etc/temporal` since that's where they end up in the dockerfiles?

Reply:
E.g. https://github.com/temporalio/docker-builds/blob/347568012222589dbe2577dcdc7f12b6855a72c2/server.Dockerfile#L64-L65

Reply:
Thanks @jonjohnsonjr, I was thinking of doing it in the main temporal-server repo? What will be better? https://github.com/wolfi-dev/os/pull/7401/files#diff-a96a0e1f38ff3cd582e50a8b59770f2cc04245eacdecea305ebad441a16d39feR73

Reply:
If we can do it here itself, while others there, I think I am okay with that, eventually I hope it won't conflict?

Reply:
And can we even do that in main temporal-server repo by including this package as runtime dep? I think this should be an environment dep for our cause?

Reply:
Having them under `/temporal-docker-builds/` doesn't give us anything, right? It's just a temporary location that could be anything?

If that's true, I think it makes sense to just put them under `/etc/temporal` so we can skip the copies in `temporal-server-oci-entrypoint`.

> And can we even do that in main temporal-server repo by including this package as runtime dep?

I think that would work, yeah.

Reply:
Yes, I guess, thanks for this suggestion, just fixed it @jonjohnsonjr !

Reply:
Just fixed this too https://github.com/wolfi-dev/os/pull/7401/commits/b27d619732be5fc789fe7b5727e77f70c4083662